src/Compilers/CSharp/Portable/Symbols/Metadata/PE/NativeIntegerTypeDecoder.cs

```diff
@@ -137,6 +139,45 @@ private PointerTypeSymbol TransformPointerType(PointerTypeSymbol type)
             return type.WithPointedAtType(TransformTypeWithAnnotations(type.PointedAtTypeWithAnnotations));
         }
 
+        private FunctionPointerTypeSymbol TransformFunctionPointerType(FunctionPointerTypeSymbol type)
+        {
+            IgnoreIndex();
+
+            var transformedReturnType = TransformTypeWithAnnotations(type.Signature.ReturnTypeWithAnnotations);
+            var transformedParameterTypes = ImmutableArray<TypeWithAnnotations>.Empty;
+            var paramsModified = false;
+
+            if (type.Signature.ParameterCount > 0)
+            {
+                var builder = ArrayBuilder<TypeWithAnnotations>.GetInstance(type.Signature.ParameterCount);
+                foreach (var param in type.Signature.Parameters)
+                {
+                    var transformedParam = TransformTypeWithAnnotations(param.TypeWithAnnotations);
+                    paramsModified = paramsModified || !transformedParam.IsSameAs(param.TypeWithAnnotations);
```

Comment:
>= paramsModified || [](start = 35, length = 19)

nit: `||=`?

Reply:
[https://github.com/dotnet/csharplang/issues/1718](https://github.com/dotnet/csharplang/issues/1718) if you want to try and convince the LDM to bring the idea back :wink:

---
In reply to: [422360892](https://github.com/dotnet/roslyn/pull/44078#discussion_r422360892) [](ancestors = 422360892)